node-red-contrib-bot-message/bot-send-card.js

```diff
@@ -538,89 +546,76 @@ const addImageToAdaptiveCard = (imageUrl, imageSize, imageHorizontalAlignment, i
-        
-        
-        reply.type = 'AdaptiveCard';
-        reply.title = title;
-        reply.version = "1.3";
-        reply.body = [{"type": "TextBlock",
-        "size": "Medium",
-        "weight": "Bolder",
-        "text": title,
-        "horizontalAlignment": "Center",
-        "wrap": true},
-        {
-            "type": "TextBlock",
-            "text": subtext,
-            "size": "Large",
-            "weight": "Bolder",
-            "wrap": true
-        },
-        {
-            "type": "Input.Text",
-            "style": "text",
-            "isMultiline": true,
-            "id": "userFeedback",
-            "placeholder": "master.placeholder",
-            "maxLength": 500,
-            "isRequired": true,
-            "label": "Feedback:",
-            "errorMessage": "master.errorMessage"
-        }];
-        reply.actions = [{
-            "type": "Action.Submit",
-            "title": "master.submit",
-            "data": {
-                "id": "feedbackButton",
-              }
-        }];
-        const textToShow = marshall(locale, subtext,  data, '');
-    }
+    formtitle = helper.getContextValue(RED, node, data, formtitle || '', config.titleformTypeAdaptiveCard || 'str');
+    formtitle = marshall(locale, formtitle,  data, '');
+    
+    
+    reply.type = 'AdaptiveCard';
+    reply.formtitle = formtitle;
+    reply.version = "1.3";
+    reply.body = [{"type": "TextBlock",
+    "size": "Medium",
+    "weight": "Bolder",
+    "text": formtitle,
+    "horizontalAlignment": "Center",
+    "wrap": true},
+    {
+        "type": "Input.Text",
+        "style": "text",
+        "isMultiline": true,
+        "id": "userFeedback",
+        "placeholder": placeHolderform,
+        "maxLength": 500,
+        "isRequired": true,
+        "label": "Feedback:",
+        "errorMessage": errorMsgform
+    }];
+    reply.actions = [{
+        "type": "Action.Submit",
+        "title": submitLabelform,
+        "data": {
+            "id": "feedbackButton",
+          }
+    }];}
+
+}
+const buildReplyformGlobalAdaptiveCard = (RED, node, locale, data, config, reply) => {
+    let formGlobaltitle = config.titleformAdaptiveCard;
+    let errorMsgformGlob = config.errorMsgformGlobalAdaptiveCard;
+    let submitLabelformGlob = config.submitLabelformGlobalAdaptiveCard;
+    let placeHolderformGlob = config.placeHolderformGlobalAdaptiveCard;
 
     if(config.sendType === 'formGlobalAdaptiveCard'){
-        title = helper.getContextValue(RED, node, data, title || '', config.titleTypeAdaptiveCard || 'str');
-        title = marshall(locale, title,  data, '');
+        formGlobaltitle = helper.getContextValue(RED, node, data, formGlobaltitle || '', config.titleformGlobalTypeAdaptiveCard || 'str');
+        formGlobaltitle = marshall(locale, formGlobaltitle,  data, '');
     
         
         reply.type = 'AdaptiveCard';
-        reply.title = title;
+        reply.formGlobaltitle = formGlobaltitle;
```

Comment:
line 601 to be removed, because there is no property 'formGlobaltitle' in the MS AdaptiveCard's schema